### ISSUE TITLE
ResNeXt-50 accuracy is 0%

### ISSUE THREAD

**Issue by gcunhase** [qubvel/classification_models#73]
## About
ResNeXt-50 accuracy is 0% instead of 77.36%. Are the pre-trained weights [here](https://github.com/qubvel/classification_models/blob/a0f006e05485a34ccf871c421279864b0ccd220b/classification_models/weights.py#L193) updated?

## Steps to reproduce
1. Install requirements:
```
pip install tensorflow-gpu==2.8.0
pip install git+https://github.com/qubvel/classification_models.git
```

2. Load the model as follows:
```
ResNeXt, preprocess_input = Classifiers.get("resnext50)
model = ResNeXt(
    include_top=True,
    input_shape=(224, 224, 3),
    weights="imagenet"
)
```

3. Use `preprocess_input` as the preprocessing function on the ImageNet validation dataset.
4. Compile the model and evaluate:
```
 model.compile(
    optimizer="sgd",
    loss=tf.keras.losses.SparseCategoricalCrossentropy(),
    metrics=["accuracy"],
)
_, model_accuracy = model.evaluate(val_batches)
```

**Note** that there are no issues with the ResNet models when I follow the above steps.

## System info
Python 3.8, Ubuntu 18.04

**Comment by gcunhase** [comment]
Is this repo still active?

**Comment by gcunhase** [comment]
Anyone still maintaining this repo?

**Comment by innat** [comment]
Are you training in the image-net dataset?
I've just tested with a small-size dataset from tensorflow-dataset and it works fine.

**Comment by gcunhase** [comment]
Yes, but I’m using ImageNet in tfrecord format. Can you please share the steps tocante it work? How you loaded the data, preprocessing, model instantiation so I can reproduce your steps on my side? Thanks!

**Comment by innat** [comment]
[HERE.](https://colab.research.google.com/drive/1lUMr2aZWiqSLAmC3T5F_8IJJFs8XS3tB?usp=sharing)

**Comment by gcunhase** [comment]
Thank you, have you tried evaluating the model directly with ImageNet (`include_top=True`)? Mine fails in that scenario.

**Comment by gcunhase** [comment]
@innat Thank you, have you tried evaluating the model directly with ImageNet (`include_top=True`)? Mine fails in that scenario.

**Comment by innat** [comment]
I don't have an ImageNet locally to test. But I did a test with `include_top=true` on the natural image. It did run but the results are kinda non-sense even on the natural images. You can see the above colab file again.

**Comment by gcunhase** [comment]
@innat Right, this is what I'm experiencing. And then when I iterate over the validation dataset, the accuracy goes to 0%.

**Comment by innat** [comment]
Oh, I see. Next, the either we can look at the source code to find anomaly or search other possible solution. Can you check other similar architecture on image-net, for example se-resnext.

**Comment by gcunhase** [comment]
@innat I'm able to get the correct accuracy with `seresnext50`...
Note I'm using pre-processing as follows due to [this issue](https://github.com/keras-team/keras/issues/12752):
```python
tf.keras.applications.imagenet_utils.preprocess_input(image, mode='torch')
```
I think that the pre-trained weights for ResNext might be incorrect!
